app/views/refinery/blog/posts/index.rss.builder

```diff
@@ -14,8 +14,10 @@ xml.rss version: "2.0", 'xmlns:atom': 'http://www.w3.org/2005/Atom', 'xmlns:medi
         xml.pubDate post.published_at.to_s(:rfc822)
         xml.link refinery.blog_post_url(post)
         xml.guid refinery.blog_post_url(post)
-        post.images.each do |image|
-          xml.media :content, url: refinery.root_url + image.url, medium: 'image'
+        if Refinery::Plugins.registered['page_images']
```

Comment:
`post.images` could come from another extension, custom code, i think we should only do:

```ruby
post.images.each do |image|
  xml.media :content, url: refinery.root_url + image.url, medium: 'image'
end if post.images.present?
```

Reply:
I thought about this, but then we do not know (for certain) that images is an array and its items have the url attribute... (or we need to more checks)

Reply:
hmm 🤔 

Is there any way that the `refinerycms-page-images` extension can add this in without overriding the entire file?

Reply:
only thing I can/could think of, is to create helper function(s) to register custom parts of the feed (e.g. image(s)) in the refinery core (so move feed feature to core). And let the different plugin(s) use those functions to build/customize the feed according to their (combined) needs...

But this is a big, complex overkill just to have a feed (IMO)

Reply:
I agree, that is quite a lot more work! I'm happy to just assume the default case of:

```ruby
post.images.each do |image|
  xml.media :content, url: refinery.root_url + image.url, medium: 'image'
end if post.respond_to?(:images)
```

Or whatever it takes to check whether we actually have an images association.

In terms of an image having a URL, I'd say that we can just assume that's the contract for a blog image? 

What do you think?

Reply:
Agree, let's not make it overly complex...!

I'll update the PR